go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/frankban/quicktest v1.9.0 // indirect
 	github.com/gardener/etcd-druid v0.3.0
 	github.com/gardener/gardener v1.6.0
+	github.com/gardener/gardener-extension-networking-calico v1.8.1-0.20200618083450-4112b60d1bac
```

Comment:
Can you reference a released version?

Reply:
no released version with this, figured this dependency after I did the release (gardener is vendored to 1.6 here which brought the generic actuator refactor in). Or I do another major release with 1 commit ...

Reply:
OK, we should update it then when a new version is available. It should not be critical as only APIs are used. OK for now then.